regolith/schemas.py

```diff
@@ -3732,11 +3802,15 @@
                 "notes": {"description": "any notes about the kickoff",
                           "required": False,
                           "type": "list"},
-                "status": {"description": f"status of the kickoff. "
-                                          f"Allowed values are {', '.join(PROJECTUM_STATUS)}",
+                "status": {"description": "status of the kickoff. "
+                                          "Allowed values are 'proposed', 'started', 'converged', 'finished', 'cancelled', 'paused'",
```

Comment:
this should link to a static variable at the top of the file, like the version you deleted.  Why did you change this?  If the list is different you can create a new variable (maybe `MILESTONE_STATUS` or sthg?)

Reply:
Done

I misunderstood that you were referring to "identifier" previously